src/components/Table/index.jsx

```diff
@@ -242,7 +237,8 @@ const Table = ({
         ...scroll,
       }}
       onChange={(pagination, _, sorter) => {
-        preserveTableStateInQuery && handleTableChange(pagination, sorter);
+        // If `handlePageChange` callback is not provided, component will handle pagination and sorting query parameters.
```

Comment:
The code is already self-explanatory. This comment does not clarify anything tricky. You can remove this comment.